docs/docs/configuration.md

```diff
@@ -285,6 +285,22 @@ This configuration can be set when the module loads or at runtime.
 
 `VKEY_MAX_ENTITY_COUNT` is 100,000.
 
+### CMD_INFO
+
+An on/off toggle for the `GRAPH.INFO` command. Disabling this command may increase performance and lower the memory usage and these are the main reasons for it to be disabled.
+
+#### Default
+
+`CMD_INFO` is `yes`.
+
+### MAX_INFO_QUERIES
+
+A limit for the number of finished queries stored for the `GRAPH.INFO PREV` subcommand.
```

Comment:
Please change `finished queries` to `previously-executed queries`

Reply:
Please also add limits (0 - 10000?)

Reply:
We did not define such a maximum value for the number of queries we can store. We assigned a uint_32 for it, but I'm not sure we want to allow that much queries to be saved.

Reply:
I think 10000. I actually mentioned "If larger than 10000 it is replaced with 10000"

Reply:
Why 10000?
Not that I have a specific problem with that number (nice and round), but I think we should think of some limit on the memory or an amount of 'relevant' queries we see right.

Reply:
We can decide based on the maximal memory size we should allow for this. 20MB? I don't know how much each entry takes. Another consideration is assuming an average throughput of e.g., 250 queries/sec for saving e.g., 60 seconds (which is 15000 in this case). @swilly22 @AviAvni what is your take here?